{
  try {
    context.getResultHandler().handleResult(context.getResult());
  }
 catch (  Exception e) {
    handleException(this,object,e);
  }
}
